### ISSUE TITLE
Problem after updating

### ISSUE THREAD

**Issue by LokiGrants** [LumaTeam/Luma3DS#277]
So, I was using luma just fine till today. Then I updated and started getting the svcBreak. Tried the troubleshoot for it and now, trying to get in by any means - even the one at troubleshooting - I get the following:

[crash_dump_00000004.zip](https://github.com/AuroraWright/Luma3DS/files/587777/crash_dump_00000004.zip)

**Comment by AuroraWright** [comment]
This is due to a pre-patched Home Menu and/or NS. Make sure you placed the loader.cxi correctly, try also to reinstall the clean titles from an older version of Luma

**Comment by LokiGrants** [comment]
May be but I have used the loader.cxi just before, I installed the clean Home Menu but couldn't install the NS cus it said it was already installed. Then it didn't want to boot anymore even in those options, that's my problem.

**Comment by LokiGrants** [comment]
Is there a way to inject the clean titles using a9lh? Cuz that's all I can use right now. Also, it is putting the error on pm, isn't it before loader.cxi is loaded?

**Comment by LokiGrants** [comment]
I found out that the ns.cia is the issue. I used a few backups - though they will always go through the same issues as the one I used - and it let's me fix the Home Menu but NS, even deleting the software with same id still gives an invalid argument that it's already installed. Do you know a way to fix it?

**Comment by pbanj** [comment]
try using dev menu instead of fbi

**Comment by LokiGrants** [comment]
I did use fbi in this case just to make easier to delete the title installed previously. But, since I couldn't get back on the thing till trying some backups and all, what I did was use a backup then ctr transfer a 9.2 version backon and inject with Decrypt9 the data of installed games and all, then the error was gone even after updating to 11.2. So, worked well in the end. Pbanj, I will backup my nand later, use another backup and check if Dev Menu can allow ns.cia to install, if it does, I will post here :) thanks.
